instrumentation/kamon-spring/src/main/scala/kamon/instrumentation/spring/client/SpringWebClientInstrumentation.scala

```diff
@@ -0,0 +1,11 @@
+package kamon.instrumentation.spring.client
+
+import kanela.agent.api.instrumentation.InstrumentationBuilder
+
+class SpringWebClientInstrumentation extends InstrumentationBuilder {
+  /*
+   * Instruments spring WebClient, mutating the request and registering callbacks.
+   */
+  onSubTypesOf("org.springframework.web.reactive.function.client.ExchangeFunction")
```

Comment:
Are there many `ExchangeFunction` implementations out there? :thinking:

Reply:
Exactly one, so I'll narrow it down